content/e20co/drone-marathon/index.md

```diff
@@ -0,0 +1,103 @@
+---
+author: "dellintosh"
+date: 2017-01-06T18:00:00-06:00
+image: "e20co/drone-marathon"
+repo: "e20co/drone-marathon"
```

Comment:
Also @bradrydzewski, we are using the `master` branch for our 0.4 version (see the `drone-0.5` branch for the new version).